### ISSUE TITLE
Revise Figma Mockup to Align with Project Proposal

### ISSUE THREAD

**Issue by gerri-barnes** [CSU-CPSC4205-2023-Summer/group_1#1]
Revise Figma Mockup to align with Pet Grooming Appointment System Project Proposal for Poochi Pet Resort. Once you have a mockup and flow workout for interaction with the application create a YouTube video to explain the user interface and flow. Attach link to revised figma mockup when complete and comment including @ to remaining group members for review.

Branding:

Figma mockup pages should include branding for Poochi Pet Resort. Here is a link to Poochi Pet Resort's website, https://poochipetresort.com/grooming/

Included Features:

●	User Registration: Pet owners can create accounts to manage their pet's grooming appointments.
●	Pet Profile Management: Users can add and update pet profiles, including pet details such as name, breed, age, and any specific grooming requirements.
●	Appointment Scheduling: Users can view available grooming slots and schedule appointments for their pets based on their preferred date and time.
●	Appointment Confirmation: Users will receive confirmation emails or notifications upon successful appointment booking.
●	Groomer Management: Admin or authorized staff can manage groomer profiles, including their availability and expertise.
●	Admin Dashboard: An administrative dashboard will be provided to manage appointments, view reports, and perform system maintenance tasks.

[Pet Grooming Appointment Web Application-Project Proposal.docx](https://github.com/CSU-CPSC4205-2023-Summer/group_1/files/11951193/Pet.Grooming.Appointment.Web.Application-Project.Proposal.docx)

[Figma .docx](https://github.com/CSU-CPSC4205-2023-Summer/group_1/files/11951273/Figma.docx)

**Comment by gerri-barnes** [comment]
@smithkayla26 - Please let me or @srenner02 know if you have any questions or need to collaborate on the Figma mockup revision. We'll review it as a group once it's complete. Thank you!

**Comment by gerri-barnes** [comment]
@smithkayla26 and @srenner02 - For the Youtube video portion of the Figma Mockup, should Kayla do the Youtube video since she's creating the it or should we do this as a group? Not sure what would be best so wanted your input. I know this was due last week but still want to ensure we have all required project deliverables.

**Comment by smithkayla26** [comment]
I can't see what you assigned. Also, my computer doesn't have any more storage, so I won't be able to screen record to make it as a YouTube video.


________________________________

**Comment by gerri-barnes** [comment]
Hi Kayla,

Can you not clean up storage space on your computer? How much storage space
does it take?

Are unable to access our project board via the following links?

https://github.com/orgs/CSU-CPSC4205-2023-Summer/projects/1

https://github.com/orgs/CSU-CPSC4205-2023-Summer/projects/1/views/1?pane=issue&itemId=32438527

**Comment by smithkayla26** [comment]
Yeah, in May I was trying to delete some files but it still didn't work. It has 212 GB.
________________________________
